test/e2e/kind-cluster.yaml

```diff
@@ -16,7 +16,12 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
-- role: control-plane
-- role: worker
-- role: worker
-- role: worker
\ No newline at end of file
+- &node
+  role: control-plane
+  image: kindest/node:v1.22.13@sha256:4904eda4d6e64b402169797805b8ec01f50133960ad6c19af45173a27eadf959
+- <<: *node
```

Comment:
I'm gonna add a comment explaining that. Since I pinned the Kubernetes version by adding the appropriate kindest image and the image should be set in all nodes, to avoid a future mistake (someone bumps the Kubernetes version in the control plane and forgets to do the same in the data plane) I used the merge key strategy.